components/_util/wave.tsx

```diff
@@ -195,7 +198,19 @@ export default class Wave extends React.Component<{ insertExtraNode?: boolean }>
     const { children } = this.props;
     this.csp = csp;
 
-    return children;
+    if (!React.isValidElement(children)) return children;
+
+    let ref: React.Ref<any> = this.containerRef;
+    if ((children as any).ref) {
```

Comment:
Use `supportRef` to check element need ref or not:
https://github.com/ant-design/ant-design/blob/0fec3aa6022c2c0ca5f11a4ff128cfa0082d848a/components/form/FormItem.tsx#L357